build-reporter-plugin.js

```diff
@@ -12,7 +12,13 @@ class BugsnagBuildReporterPlugin {
     const plugin = (compilation, cb) => {
       const stats = compilation.getStats()
       if (stats.hasErrors()) return cb(null)
-      reportBuild(this.build, this.options)
+      const options = Object.assign(this.options,
+        !this.options.logger && compiler.getInfrastructureLogger
+          ? { logger: compiler.getInfrastructureLogger('BugsnagBuildReporterPlugin') }
+          : {}
+      )
+
+      reportBuild(this.build, options)
```

Comment:
This means that for webpack >= 4.37 the logLevel documented option does nothing because the built-in logger is not used. This also applies (as it did before) if you pass a custom logger. Not sure if this is worth documenting?

Reply:
I think that's fine. We should update the docs to say which cases it works in.